src/waveform/renderers/glslwaveformrenderersignal.cpp

```diff
@@ -1,6 +1,11 @@
 #include "waveform/renderers/glslwaveformrenderersignal.h"
 #if !defined(QT_NO_OPENGL) && !defined(QT_OPENGL_ES_2)
 
+#ifdef MIXXX_USE_QOPENGL
+#include <QOpenGLFramebufferObject>
+#include <QOpenGLShaderProgram>
+#else
+#endif
```

Comment:
This #else / #endif  looks placed wrong here?

Reply:
absolutely, good catch